etc/nginx/snippets/nginx.conf

```diff
@@ -0,0 +1,27 @@
+# Used with a docker container for localised testing.
```

Comment:
Presumably the nginx.conf is just for your local testing? I don't think our deploy will do anything with it.

Perhaps rename to `nginx.conf.example` and extent the comment to explain what it's used for.